lib/cosmos/tools/script_runner/script_runner_frame.rb

```diff
@@ -999,9 +1006,32 @@ def show_debug
             @debug_text.setPlainText("")
           end
         end
-
         @debug_frame.addWidget(@debug_text)
 
+        @backtrace_button = Qt::PushButton.new('Error Backtrace')
```

Comment:
I think this is useful in the menus, instead of a button, because it is useful even in production and when the debug mode is not enabled.

Reply:
Seems like this only applies while running?

Reply:
No, it is also valid when not running.   For example on a logic error when the script stops.   Also, I think it should toggle showing the backtrace. Right now there is no easy way to turn it off